lib/jsdom/living/events/EventHandler.webidl

```diff
@@ -0,0 +1,20 @@
+// https://html.spec.whatwg.org/multipage/webappapis.html#eventhandlernonnull
```

Comment:
Hmm, maybe name this file event-handlers.webidl since it contains more than just EventHandler and its supporting definitions?